File: code/community/1302075420622983179/user-removal-api-keys-impact.go
package main

import (
	"fmt"
	"net/http"
	"os"
)

func main() {
	// Get your Deepgram API key from the environment variables
	apiKey := os.Getenv("DEEPGRAM_API_KEY")

	// Deepgram API endpoint
	url := "https://api.deepgram.com/v1/listen"

	// Create a new HTTP request
	req, err := http.NewRequest("POST", url, nil)
	if err != nil {
		fmt.Println("Error creating request:", err)
		return
	}

	// Add the Authorization header
	req.Header.Add("Authorization", "Bearer "+apiKey)

	// Execute the request
	client := &http.Client{}
	resp, err := client.Do(req)
	if err != nil {
		fmt.Println("Error executing request:", err)
		return
	}
	defer resp.Body.Close()

	// Check the status and print the response
	fmt.Println("Response status:", resp.Status)
}

// NOTE: Before running, ensure you have set the DEEPGRAM_API_KEY environment variable.

File: code/community/1302075420622983179/user-removal-api-keys-impact.js
const axios = require('axios');

// Get your Deepgram API key from the environment variables
const apiKey = process.env.DEEPGRAM_API_KEY;

// Deepgram API endpoint
const url = 'https://api.deepgram.com/v1/listen';

(async () => {
  try {
    // Make the HTTP POST request
    const response = await axios.post(url, null, {
      headers: {
        'Authorization': `Bearer ${apiKey}`
      }
    });

    // Check the status and print the response data
    if (response.status === 200) {
      console.log('Response:', response.data);
    } else {
      console.error('Request failed with status:', response.status);
    }
  } catch (error) {
    console.error('Error making request:', error);
  }
})();

// NOTE: Before running, ensure you have set the DEEPGRAM_API_KEY environment variable.